deployment/README.md

```diff
@@ -0,0 +1,6 @@
+This serves as an example for how to deploy everything in a single box. This is far
```

Comment:
do we want to call it single box? How about single container? more understandable I feel

Reply:
Changed to "machine", probably a slightly more understandable term